Automattic/wp-calypso
# Woo Express: Add plan features and feature groups

Related to #75383

## Proposed Changes

This adds all of the Woo Express-specific plan features and feature groups.

This is part of several PRs related to plan features and the plan comparison grid:
* #75918 (this PR) which adds all of the plan features and groups.
* #75769 which enables the plan comparison grid for Woo Express.
* #75883 which adds footnotes to the plan comparison grid.

## Testing Instructions



This Slack conversation (p1681499986012049-slack-C0Q664T29) has a link to a spreadsheet with all of the features.

 * Inspect the code and verify that the correct features exist and are listed under the correct groups.


## Pre-merge Checklist
